### PR TITLE
que3.4

### PR DESCRIPTION
**Pull request by ashok-choudhary-0**

### CODE REVIEW

File: controllers/userController.js [hunk 1]
```diff
@@ -183,11 +185,22 @@ const verifyResetPasswordToken = async (req, res) => {
...
   } catch (err) {
     res.status(500).send(err)
   }
-
 }
-module.exports = { userRegister, login, getUserData, deleteUserData, limitUsersData, userAddress, deleteUserAddresses, forgotPassword, verifyResetPasswordToken }
+const uploadProfile = async (req, res) => {
+  const username = req.params.username;
+  try {
+    const userData = await userModel.findOne({ where: { username } })
```
Comment (amrit56789) [inline controllers/userController.js]: Here variable name here you are finding a user so take a variable name relatable.

File: controllers/userController.js [hunk 2]
```diff
@@ -183,11 +185,22 @@ const verifyResetPasswordToken = async (req, res) => {
...
     }
-
-
   } catch (err) {
     res.status(500).send(err)
   }
-
 }
-module.exports = { userRegister, login, getUserData, deleteUserData, limitUsersData, userAddress, deleteUserAddresses, forgotPassword, verifyResetPasswordToken }
+const uploadProfile = async (req, res) => {
```
Comment (amrit56789) [inline controllers/userController.js]: addUserImage

File: controllers/userController.js [hunk 3]
```diff
@@ -183,11 +184,22 @@ const verifyResetPasswordToken = async (req, res) => {
...
-
-
   } catch (err) {
     res.status(500).send(err)
   }
-
 }
-module.exports = { userRegister, login, getUserData, deleteUserData, limitUsersData, userAddress, deleteUserAddresses, forgotPassword, verifyResetPasswordToken }
+const addUserProfileImage = async (req, res) => {
+  const username = req.params.username;
```
Comment (meanstack9716) [inline controllers/userController.js]: why ask the user to give you the user name? You can get the user's id  from the token, use that @ashok-choudhary-0

File: controllers/userController.js [hunk 4]
```diff
@@ -183,11 +184,22 @@ const verifyResetPasswordToken = async (req, res) => {
...
+  try {
+    const findUser = await userModel.findOne({ where: { username } })
```
Comment (meanstack9716) [inline controllers/userController.js]: should be by id @ashok-choudhary-0

**Review by meanstack9716 — CHANGES_REQUESTED** [review]

**Pull request merged**